web/src/inbox_ui.js

```diff
@@ -585,41 +593,47 @@ function set_list_focus(input_key) {
     }
 
     const $row_to_focus = $($all_rows.get(row_focus));
-    const $cols_to_focus = $row_to_focus.find("[tabindex=0]");
+    // This includes a fake collapse button for `inbox-row`.
+    const $cols_to_focus = [$row_to_focus, ...$row_to_focus.find("[tabindex=0]")];
```

Comment:
Hmm, but does this have a duplicate for `inbox-header`? I'm not sure `...` avoids that in a list context.

Reply:
Since we are using `.find` it doesn't include `$row_to_focus` which is either `inbox-row` or `inbox-header`.

Reply:
Ahh, OK, yes that works.